packages/math-input/src/components/many-keypad-button.tsx

```diff
@@ -3,30 +3,30 @@
  * 'default' symbol.
  */
 
-import PropTypes from "prop-types";
 import * as React from "react";
 
 import {KeyTypes} from "../consts";
 import KeyConfigs from "../data/key-configs";
 import Keys from "../data/keys";
 
 import EmptyKeypadButton from "./empty-keypad-button";
-import {keyIdPropType} from "./prop-types";
 import TouchableKeypadButton from "./touchable-keypad-button";
 
-class ManyKeypadButton extends React.Component {
-    static propTypes = {
-        keys: PropTypes.arrayOf(keyIdPropType).isRequired,
-    };
+import type {Key} from "../data/keys";
 
+type Props = {
+    keys?: Array<Key>;
+};
+
+class ManyKeypadButton extends React.Component<Props> {
     render() {
-        const {keys, ...rest} = this.props;
+        const {keys = [], ...rest} = this.props;
 
         // If we have no extra symbols, render an empty button. If we have just
         // one, render a standard button. Otherwise, capture them all in a
         // single button.
         if (keys.length === 0) {
-            return <EmptyKeypadButton {...rest} />;
+            return <EmptyKeypadButton />;
```

Comment:
Is this because `EmptyKeypadButton` doesn't take any props?

Reply:
Right, with the exception of ReduxProps